.travis.yml

```diff
@@ -11,6 +11,8 @@ addons:
     packages:
     - icnsutils
     - graphicsmagick
+before_script:
+- export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start || true
```

Comment:
`|| true` was to skip OSX, since Xfvb doesn't happen on there. Buut that's definitely a bad idea in case it fails on Linux, so I've made it better. 👌